src/components/Persona/Persona.tsx

```diff
@@ -60,15 +68,8 @@ export class Persona extends React.Component<IPersonaProps, any> {
 
         { presenceElement }
 
-        { !hidePersonaDetails && (
-          <div className='ms-Persona-details'>
-            <div className='ms-Persona-primaryText'>{ primaryText }</div>
-            { secondaryText ? (
-              <div className='ms-Persona-secondaryText'>{ secondaryText }</div>
-            ) : (null) }
-            <div className='ms-Persona-tertiaryText'>{ tertiaryText }</div>
-            <div className='ms-Persona-optionalText'>{ optionalText }</div>
-          </div>) }
+        { !hidePersonaDetails && personaDetails }
+        { (hidePersonaDetails && (size === PersonaSize.tiny) && personaDetails) }
```

Comment:
How about something like this instead:

`(!hidePersonaDetails || (size === PersonaSize.tiny)) && personaDetails`